templ_attr.cif

```diff
@@ -293,8 +293,9 @@ save_site_symmetry
      The character string n_pqr is composed as follows:
 
          n refers to the symmetry operation that is applied to the
-         coordinates stored in _atom_site.fract_xyz. It must match a
-         number given in _symmetry_equiv.pos_site_id.
+         coordinates stored in _atom_site.fract_xyz. It must match
+         a number given in _space_group_symop.id (or one of its
+         aliases).
```

Comment:
The original improvement suggested in issue #295 also included `_symmetry_equiv_pos_site_id` as an example of the an alias that one could reasonably expect to encounter in legacy CIF files. Without it, the text in the parentheses "`(or one of its aliases)`" does not convey any specific information since the same same general approach (replacement) applies to all items that have aliases.

I would probably either introduce an example of a specific alias or remove the parentheses altogether.

Reply:
Example introduced. I had originally read James' comment differently; I now concur with you.